### PR TITLE
fix(ListingPage): Adjust ListingPage to address some layout issues

### PR DESCRIPTION
**Pull request by bdauria**
This PR addresses a few layout issues encountered after migrating the Events view page to the ListingPage.

### CODE REVIEW

File: src/ListingPage/SlidePanel.tsx [hunk 1]
```diff
@@ -1,32 +1,70 @@
...
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    height: '100%',
+    display: 'grid',
+    gridTemplate: 'auto auto 1fr / 1fr',
+  },
+  header: {
+    gridArea: '1 / 1 / 2 / 1',
+    padding: 8,
```
Comment (kduret) [inline src/ListingPage/SlidePanel.tsx:22]:
```suggestion
    padding: theme.spacing(2),
```

File: src/ListingPage/SlidePanel.tsx [hunk 2]
```diff
@@ -38,16 +76,32 @@ const SlidePanel = ({ header, content }: SlidePanelProps): JSX.Element => {
...
+          <AppBar position="static" color="default">
+            {!isEmpty(tabs) && (
+              <Tabs
+                variant="fullWidth"
+                value={selectedTabId}
+                indicatorColor="primary"
+                textColor="primary"
+                onChange={onTabSelect}
+              >
+                {tabs.map((tab) => tab)}
```
Comment (kduret) [inline src/ListingPage/SlidePanel.tsx:96]: No need to add key ?

Comment (bdauria) [reply src/ListingPage/SlidePanel.tsx:96]: There is going to be a linting error if the Component using the SlidePanel dosen't provide keys to the given tabs.

File: src/ListingPage/index.stories.tsx [hunk 3]
```diff
@@ -241,84 +257,90 @@ interface PanelProps {
...
-      filtersExpandable
-      filters={
-        <FiltersSummaryWithOpenSlidePanelButton onOpen={() => setOpen(true)} />
-      }
-      expandableFilters={<FiltersDetails />}
-      slidePanel={<DetailsPanel onClose={() => setOpen(false)} />}
-      slidePanelIntegrate
+      filters={<ExpandableFiltersWithOpenButton onOpen={() => setOpen(true)} />}
+      panel={<DetailsPanel onClose={() => setOpen(false)} />}
+      panelFixed={false}
```
Comment (Thebarda) [inline src/ListingPage/index.stories.tsx:343]:
```suggestion
      panelFixed
```
A story with fixed panel is missing

File: src/index.ts [hunk 4]
```diff
@@ -49,7 +49,8 @@ export {
 } from './Listing/models';
 
 export { default as ListingPage } from './ListingPage';
-export { default as SlidePanel } from './ListingPage/SlidePanel';
+export { default as FilteBar } from './ListingPage/Filters';
```
Comment (Thebarda) [inline src/index.ts:52]:
```suggestion
export { default as Filters } from './ListingPage/Filters';
```

File: src/ListingPage/index.stories.tsx (hunk 3 above)
Comment (bdauria) [reply src/ListingPage/index.stories.tsx]: > I see a regression about Listing Page resizing when expanding or collapsing filters
> ListingPage is no longer resizing

Can you elaborate on the regression? it seems to work as before for me

Comment (Thebarda) [reply src/ListingPage/index.stories.tsx]: Before Listing Page height was resized 'ouf of the box' (see : https://github.com/centreon/centreon-ui/blob/master/src/ListingPage/index.tsx#L60)
I will check on AutoDiscovery if it actually works

File: src/Panel/index.tsx [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  id: number;
+}
+
+interface Props {
+  header: React.ReactElement;
+  selectedTab: React.ReactElement;
+  tabs?: Array<JSX.Element>;
+  selectedTabId?: number;
+  onTabSelect?: (event, id: number) => void;
+  onClose: () => void;
```
Comment (Thebarda) [inline src/Panel/index.tsx:64]: `onClose` is not required. For example, in hosts listing, we always display mappers panel

**Pull request merged**